.github/workflows/tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         run: |
           pip install -U pip
           pip install -r requirements.txt
-          pip install -r requirements/testing_without_asyncio.txt
```

Comment:
Can you revert this change? This part intentionally does not have aiohttp to detect unexectedly having aiohttp depedency to the core part  (as the following name "Run tests without aiohttp" indicate)

Reply:
I've reverted this change but `aoihttp` will be required to run `pytest tests/adapter_tests/socket_mode/`, we can try using another synchronous library or move the CI step `Run tests for Socket Mode adapters` after `Run tests for HTTP Mode adapters (asyncio-based libraries)` let me know what you think

Reply:
> pytest tests/adapter_tests/socket_mode/

Sorry, I should have clearly mentioned this but adding aiohttp for these tests is totally. My concern was about tests under slack_sdk etc.

Reply:
Ohh I see, I've update the ci tests and dependencies to make them pass, let me know if we prefer a sync approach to this?

Reply:
Thank you! The only risk is that we may unintentionally add aiohttp imports to the non-asyncio Socket Mode adapters. Thus, making the changes only to the adapter tests should be safe enough. We can detect such in code reviews.